app/pages/settings/settings.tsx

```diff
@@ -54,21 +54,12 @@ export const Settings = () => {
               node={node}
               activeNode={selectedNodeApi}
               onChange={nodeId => dispatch(setActiveStacksNode(nodeId))}
-              onEdit={() => {
-                setOperation('update');
-                setNodeModalOpen(true);
-              }}
+              onEdit={() => (setOperation('update'), setNodeModalOpen(true))}
```

Comment:
Interesting pattern!

Reply:
I'd much rather remove the need for this `operation` toggle, but haven't figured out another way to infer the type of operation